admin/admin-filters-term.php

```diff
@@ -658,12 +658,18 @@ public function get_inserted_term_language( $lang ) {
 		}
 
 		// Special cases for default categories as the select is disabled.
-		elseif ( ! empty( $_POST['tag_ID'] ) && in_array( intval( get_option( 'default_category' ) ), $this->model->term->get_translations( (int) $_POST['tag_ID'] ), true ) ) { // phpcs:ignore WordPress.Security.NonceVerification
-			$lang = $this->model->term->get_language( (int) $_POST['tag_ID'] ); // phpcs:ignore WordPress.Security.NonceVerification
+		elseif ( ! empty( $_POST['tag_ID'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+			$default_term = get_option( 'default_category' );
+			if ( is_int( $default_term ) && in_array( $default_term, $this->model->term->get_translations( (int) $_POST['tag_ID'] ), true ) ) { // phpcs:ignore WordPress.Security.NonceVerification
```

Comment:
`is_int( $default_term )` so we're neglecting the case where the option is filtered and the returned value is a numeric string?

Reply:
You are right. I changed this in https://github.com/polylang/polylang/pull/1278/commits/4ae13f8681c683073e39548fac36964e3f4aff1a